lib/routes/jlu/phy/rcpy/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Route } from '@/types';
+import { handler } from '../handler';
+
+export const route: Route = {
+    path: '/phy/rcpy/:category/:column',
+    categories: ['university'],
+    example: '/jlu/phy/rcpy/bksjy/BKZX',
+    radar: [
+        {
+            source: ['phy.jlu.edu.cn/rcpy/bksjy/BKZX'],
+        },
+    ],
+    name: '吉林大学物理学院 - 本科生培养',
```

Comment:
No need to repeat namespace name in route name
```suggestion
    name: '物理学院 - 本科生培养',
```